lib/src/revset.rs

```diff
@@ -1086,6 +1106,29 @@ fn expect_one_optional_argument<'i>(
     }
 }
 
+fn expect_two_optional_argument<'i>(
+    name: &str,
+    arguments_pair: Pair<'i, Rule>,
+) -> Result<(Option<Pair<'i, Rule>>, Option<Pair<'i, Rule>>), RevsetParseError> {
+    let span = arguments_pair.as_span();
+    let mut argument_pairs = arguments_pair.into_inner().fuse();
+    if let (opt_arg1, opt_arg2, None) = (
+        argument_pairs.next(),
+        argument_pairs.next(),
+        argument_pairs.next(),
+    ) {
+        Ok((opt_arg1, opt_arg2))
+    } else {
+        Err(RevsetParseError::with_span(
+            RevsetParseErrorKind::InvalidFunctionArguments {
+                name: name.to_owned(),
+                message: "Expected 0 or 1 or 2 arguments".to_string(),
```

Comment:
Nit: `0, 1, or 2 arguments` OR `0 to 2 arguments`?

Reply:
Done.